modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java

```diff
@@ -110,6 +120,7 @@ private DDMFormEvaluatorEvaluateRequest() {
 	private DDMForm _ddmForm;
 	private DDMFormLayout _ddmFormLayout;
 	private DDMFormValues _ddmFormValues;
+	private boolean _editing;
```

Comment:
@marcosapmf, what exactly is this `editing` attribute referring?

Reply:
@marcosapmf, maybe we should rename it to `_userEditing`, after all the user is editing the field and the field is being edited, what do you think?

Reply:
@natocesarrego it storages the information if the user is editing a field. Maybe we should rename it to `_editingField`. What do you think?